docs/contributors/code/release.md

```diff
@@ -130,15 +130,31 @@ If you decide that the fixes deserve another release candidate before the stable
+As you proceed with the following process, it's worth bearing in mind that such point releases are not created as branches in their own right (e.g. `release/12.5.0`) but are simply [tags](https://github.com/WordPress/gutenberg/releases/tag/v12.5.1).
 
-**Please note** however, that if the previous release was an _RC_ (e.g. `x.y.0-rc.1`), the workflow will try to release the next major _stable_ version. Therefore, should you wish to release a point release for a stable version of the Plugin (e.g. `10.1.0`) but there is an _RC already published_ for the _next_ version of the Plugin (e.g. `10.2.0-rc.1`) you will need to manually select the stable version's release branch when creating the release.
+The method for point releases is nearly identical to the main Plugin release process (see above) but has some notable exceptions. Please make sure to read _the whole_ of this guide before proceeding.
 
-To do this, when running the Workflow, select the appropriate `release/` branch from the `Use workflow from` dropdown (e.g. `release/10.1`) and specify `stable` in the text input field.
+#### Updating the release branch
 
-Please note you **cannot create point releases for previous stable releases once a more recent stable release has been published** as this would require significant changes to how we upload plugin versions to the WP.org plugin SVN repo).
+The point release should only contain the _specific commits_ required. To do this you should checkout the previous _minor_ stable (i.e. non-RC) release branch (e.g. `release/12.5`) locally and then cherry pick any commits that you require into that branch.
+
+Once you have the stable release branch in order you can _push it to Github_ and continue with the release process using the Github website GUI.
+
+#### Running the point release
+
+![Run workflow dropdown](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/contributors/code/workflow-dispatch-banner.png)
+
+Go to Gutenberg's GitHub repository's Actions tab, and locate the ["Build Gutenberg Plugin Zip" action](https://github.com/WordPress/gutenberg/actions/workflows/build-plugin-zip.yml). You should now _carefully_ choose the next action based on information about the current Plugin release version:
+
+_If_ the previous release version was **stable** (`X.Y.Z` - e.g. `12.5.0`, `12.5.1` .etc) leave the the `Use workflow from` field as `trunk` and then specify `stable` in the text input field. The workflow will automatically create a point release, with z incremented (`x.y.(z+1)`) as required.
+
+_If_ however, the previous release was an **RC** (e.g. `X.Y.0-rc.1`) you will need to _manually_ select the _stable version's release branch_ (e.g. `12.5.0`) when creating the release. Failure to do this will cause the workflow to release the next major _stable_ version which is not what you want.
```

Comment:
Doc looks great to me, this is the one section I'm not totally clear on. It's very clear on what to do when the previous version was an RC, but I'm completely sure what to expect if I were to fail to select the stable version's release branch (e.g. `12.5.0`).

> Failure to do this will cause the workflow to release the next major _stable_ version which is not what you want.

Does this mean that failing to select `v12.5.0` would ultimately release `12.6.0`, rather than the `12.5.1` patch that the contributor was hoping for? If so, I'd recommend moving the emphasis in the above quote from the word _stable_ to _major_ :)

Reply:
Thanks. Ok I've added in an example to illustrate this clearly. Yes if there is an RC then the workflow will try and release the stable version for that RC. So if you're RC is `12.6.0-rc.1` then running the workflow without taking due care would result in the release of `12.6.0` (stable).